lib/Account.php

```diff
@@ -42,24 +42,119 @@ class Account extends ApiResource
     use ApiOperations\NestedResource;
     use ApiOperations\Update;
 
+    const AU_BECS_DEBIT_PAYMENTS_ACTIVE = 'active';
```

Comment:
I don't think we should do those this way since they are just identical. If I had done this right originally, it should be a list of `STATUS_*` constants inside the `Capability` class instead right?